misc/e2e-aws.sh

```diff
@@ -167,7 +167,7 @@ stage-vault() {
     # Download Ansible playbooks for Vault bootstrap.
     local tmp=$(mktemp -d)
     cd $tmp
-    git clone --depth 1 --quiet https://taylorbot:${E2E_GITHUB_TOKEN}@github.com/giantswarm/hive.git
+    git clone --depth 1 --quiet https://taylorbot:${E2E_GITHUB_TOKEN}@github.com/giantswarm/hive.git -b vault_0.10.3
```

Comment:
we should remove  cloning from branch after stuff is tested and merged, lets use master only

Reply:
sure, was just pushing last branch